docs/advanced-topics/best-practices/best-practices-secure-fair-workflows.rst

```diff
@@ -251,4 +256,6 @@ Do not use untagged or “latest”.
 			- Any available cost or benchmarking information. 
 			- How to cite the use of your workflow (including references for the original software authors). 
 
+- Note: Documentation can be housed at either or both, the metadata section of the workflow file and/or the GitHub README document. On Dockstore, if a description is provided in the metadata section, it will be displayed by default in the INFO section. If a metadata section is missing, Dockstore will pull information from the README file on the original GitHub repo.
```

Comment:
Suggestions:

"it will be displayed by default in the INFO section" -> it will be displayed on the INFO tab

"Dockstore will pull information from the README file on the original GitHub repo" -> "Dockstore will display the README on the INFO tab"

Reply:
Also "metadata section" is arguably somewhat WDL-specific, but I don't feel too strongly

Reply:
Could link to the stable versions of  https://docs.dockstore.org/en/develop/advanced-topics/best-practices/best-practices.html?highlight=metadata#authorship-metadata and https://docs.dockstore.org/en/develop/advanced-topics/best-practices/best-practices.html?highlight=metadata#authorship-metadata

Reply:
@avani-k I think Denis is suggesting you add these linked docs by referencing their location in the Dockstore documentation repo. For example, :doc:`Authorship-metadata </advanced-topics/best-practices/authorship-metadata>`. Double check that I got this correct.